### PR TITLE
Improve continuous integration

### PR DESCRIPTION
**Pull request by dylandoamaral**
I put here several ideas to improve continuous integration of `zio-spark`, will change this PR according to your choices:
- Enable scalafix using the same configuration as [zio](https://github.com/zio/zio/blob/master/.scalafix.conf) => [organize-import](https://github.com/liancheng/scalafix-organize-imports#sbt) could be a good lib too, tell me if you want me to add it in the PR.
- [Scala Steward](https://github.com/scala-steward-org/scala-steward) also can help zio-spark to upgrade dependencies. Just add this repository inside [this file](https://github.com/scala-steward-org/repos/blob/master/repos-github.md). That's also a good idea to add a mergify rule for scala steward to automerge his own PR using this kind of configuration: ([1](https://github.com/dylandoamaral/zinteract/blob/master/.mergify.yml), [2](https://github.com/zio/zio/blob/master/.mergify.yml)).
- Add badges to the readme for coverage, integration etc.

Have a nice ziohackathon !

### CODE REVIEW

**Comment by ahoy-jon** [comment]
@dylandoamaral great ideas, you can complete the PR to the max!

Thanks for taking a look at the project!

**Comment by ahoy-jon** [comment]
It is fixed for good ! I review it in details, and i merge !

File: .circleci/config.yml [hunk 1]
```diff
@@ -81,7 +81,5 @@ jobs:
     <<: *environment
     working_directory: ~/repo
     steps:
-      steps:
-        - build_project:
-            project_name: zio-spark
-  
+      - build_project:
+          project_name: "zio-spark"
```
Comment (ahoy-jon) [inline .circleci/config.yml:85]: @dylandoamaral c'est correct le YAML ici ? j'ai du mal à voir si c'est bon.

**Review by ahoy-jon — APPROVED** [review]
C'est top, à part le bout de YAML où je ne sais pas si c'est le bon niveau d'indentation ... ! 💯

File: .circleci/config.yml (hunk 1 above)
Comment (dylandoamaral) [reply .circleci/config.yml:85]: Yes il y avait un `steps` de trop.

**Comment by dylandoamaral** [comment]
Closes #2, I intergated scaluzzi, didn't know how to configure it. Can't find any documentations :/.

**Pull request merged**